src/installation.rst

```diff
@@ -240,7 +240,7 @@ Configuration
 Cylc uses "sane and safe" defaults and is suitable for use "out of the box".
 However, many things may need to be configured, e.g:
 
-* Job hosts
+* Job hosts, see :ref:`AdminGuide.PlatformConfigs`
 * Communication methods
 * User/Site preferences
```

Comment:
Comms methods are part of platforms and user/site preferences is a bit generic.
I think there are other sections it would be helpful to link to.
How about (syntax not checked):
```
* :ref:`AdminGuide.PlatformConfigs` (jobs hosts, runners, etc)
* :ref `Submitting Workflows To a Pool Of Hosts <Scheduler hosts>`
* :ref `Event Handling <Default event handlers>`
```